boa/src/syntax/parser/expression/assignment_operator/mod.rs

```diff
@@ -0,0 +1,122 @@
+//! Assignment operator parsing.
```

Comment:
Shouldn't this be `assignment_expression` instead of `assignment_operator`?

Maybe we could shortened down to just `assignment`, since it's in `expression` module.